packages/client/jest.config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
     '<rootDir>/src/__tests__/integration/happy/exhaustive-schema/generated-dmmf.ts',
     '<rootDir>/src/__tests__/integration/happy/exhaustive-schema-mongo/generated-dmmf.ts',
     '__helpers__/',
+    'utils/',
```

Comment:
Is that to ignore this?
`packages/client/src/__tests__/integration/happy/composites-mongo/utils/build-data/commentOptionalPropDataA.ts`

Could you narrow it, since all utils folders everywhere will be ignored (and then if a utils folder has a test it will also get ignored?)